observer/observer_test.go

```diff
@@ -247,3 +248,50 @@ func Test_inferUtxoValue(t *testing.T) {
 		})
 	}
 }
+
+func TestGetInterval(t *testing.T) {
+	min, _ := time.ParseDuration("2s")
+	max, _ := time.ParseDuration("30s")
+	type args struct {
+		blockTime   int
+		minInterval time.Duration
+		maxInterval time.Duration
+	}
+	tests := []struct {
+		name string
+		args args
+		want time.Duration
+	}{
+		{
+			"test minimum",
+			args{
+				blockTime:   100,
```

Comment:
why it's called correctly here as blocktime and not as here: https://github.com/trustwallet/blockatlas/pull/881/commits/496e331b5d395e6eeb8bfc5d7ddf35905c04a36c#diff-483a22458f1a7ae9c358127c7c4f8d7bR112 ?